FAQ/supplychain.md

```diff
@@ -0,0 +1,6 @@
+## **Q: How does OSIM define "supply chain"?**
+**A:** 
+In the context of OSIM, a supply chain is the network of all the individuals, organizations, resources, activities and technology involved in the creation, sale and delivery of an artifact to the end user.
```

Comment:
How about "that work together to build, deliver, and operate software" instead of "involved in the creation, sale and delivery of an artifact"? I'm hesitant about "sale" and in some cases one supply chain may be a sub-part of another, in which case the smaller one may not actually involve end users.

Reply:
What about "that work together to build, offer, deliver, and operate software".
The offer could be an open source grant of license or the sale in commercial and mean enough in free and open source context.

Reply:
In the context of OSIM, a supply chain is the network of all the individuals, organizations, resources, activities and technology involved in the  creation, packaging, offer, license, delivery, operation, and subsequent lifecycle of an artifact.

Reply:
and make consistent with OSIM charter